src/poliastro/core/events.py

```diff
@@ -84,3 +84,55 @@ def line_of_sight(r1, r2, R):
     theta_2 = np.arccos(R / r2_norm)
 
     return (theta_1 + theta_2) - theta
+
+
+@jit
+def elevation_function(k, u_, phi, theta, R, R_p, H):
+    """Calculates a continuous visibility function for the visibility
+    of a satellite over a ground station.
+
+    Parameters
+    ----------
+    k: float
+        Standard gravitational parameter (km^3 / s^2).
+    u_: ~numpy.array
+        Satellite position and velocity vector with respect to the central attractor.
+    phi: float
+        Geodetic Latitude of the station.
+    theta: float
+        Local sidereal time at a particular instant.
+    R: float
+        Equatorial radius of the central attractor.
+    R_p: float
+        Polar radius of the central attractor.
+    H: float
+        Elevation, above the ellipsoidal surface.
+
+    """
+    ecc = np.sqrt(1 - (R_p / R) ** 2)
+    denom = np.sqrt(1 - ecc**2 * np.sin(phi) ** 2)
+    g1 = H + (R / denom)
+    g2 = H + (1 - ecc**2) * R / denom
+    station_coords = np.array(
```

Comment:
Wondering if we have these conversion functions somewhere already?

Reply:
That line performs a transformation between geocentric equatorial coordinates to the topocentric horizon coordinate system.

Using `astropy`, we could define [`AltAz`](https://docs.astropy.org/en/stable/api/astropy.coordinates.AltAz.html) and use something like `transform_to` to do the transformation.

As decided, keeping performance in mind, we could add our own implementations for that.